verticapy/core/vdataframe/base.py

```diff
@@ -324,7 +326,7 @@ def _from_object(
                 tb_final[col] = tb[col]
             tb = TableSample(tb_final)
 
-        return self.__init__(input_relation=tb.to_sql())
```

Comment:
`return` is really redundant when `__init__` is not supposed to return anything.